modal/cli/launch.py

```diff
@@ -16,22 +18,49 @@
 )
 
 
-def _launch_program(name: str) -> None:
+def _launch_program(name: str, args) -> None:
     contents = (Path(__file__).parent / "programs" / name).read_text()
+    contents = contents.replace("args: Any = {}", f"args: Any = {json.dumps(args)}")
```

Comment:
Sorry for the awful code, I think it will take a while for me to figure out how to programmatically run the modal app from within the CLI but I'll do it soon 😬